dispatch/BESStreamResponseHandler.h

```diff
@@ -22,7 +22,7 @@
 //
 // You can contact University Corporation for Atmospheric Research at
 // 3080 Center Green Drive, Boulder, CO 80301
- 
+
```

Comment:
Why is this blank line not a comment like the rest of the blank lines in the copyright preamble?